modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/java/com/liferay/portal/search/tuning/rankings/web/internal/util/RankingUtil.java

```diff
@@ -37,4 +37,18 @@ public static Collection<String> getQueryStrings(
 		return ListUtil.sort(new ArrayList<>(queryStrings));
 	}
 
+	public static String getScope(
```

Comment:
what do these strings correspond to?
seems like Sites shouldnt be plural, and just Site?
i know there has been talk of switching "Everything" to "All Sites" in search bar scope,  maybe we should start that pattern here?

Reply:
also, i dont really understand whats going on here... wouldnt the scope actually be the groupExternalReferenceCode or sxpBlueprintExternalReferenceCode? 

storing these generic strings seems redundant and ambiguous...